src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration.mdx

```diff
@@ -872,18 +868,211 @@ To set up the modules:
+
+       ### Create the namespace, persistent volume, and persistent volume claim
+
+       Make sure you've already set up your EFS filesystem and installed the [EFS CSI driver](https://github.com/kubernetes-sigs/aws-efs-csi-driver) on your cluster. You will also need your EFS filesystem ID for the PV's `spec.csi.volumeHandle`.
+      
+       ```sh
+       kubectl apply -f - <<EOF
+       apiVersion: v1
+       kind: Namespace
+       metadata:
+         name: newrelic
+      
+       ---
+       kind: StorageClass
+       apiVersion: storage.k8s.io/v1
+       metadata:
+         name: efs-sc
+       provisioner: efs.csi.aws.com
+      
+       ---
+       apiVersion: v1
+       kind: PersistentVolume
+       metadata:
+         name: custom-modules-pvc
+       spec:
+         capacity:
+           storage: 5Gi
+         volumeMode: Filesystem
+         accessModes:
+           - ReadWriteMany
+         persistentVolumeReclaimPolicy: Retain
+         storageClassName: efs-sc
+         csi:
+           driver: efs.csi.aws.com
+           volumeHandle: <your-efs-filesystem-id>
+      
+       ---
+       apiVersion: v1
+       kind: PersistentVolumeClaim
+       metadata:
+         name: custom-modules-pvc
+         namespace: newrelic
+       spec:
+         accessModes:
+           - ReadWriteMany
+         storageClassName: efs-sc
+         resources:
+           requests:
+             storage: 5Gi
+       EOF
+       ```
+      
+       Switch to the `newrelic` namespace in your `~/.kube/config`.
+
+       ```sh
+       kubectl config get-contexts
+       kubectl config set-context <your-context> --namespace=newrelic
+       kubectl config view --minify | grep namespace:
+       ```
+
+       At this point, the PVC should be bound to the PV with RWX access mode.
+
+       ```sh
+       $ kubectl get pv,pvc
+       NAME                                  CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                         STORAGECLASS   VOLUMEATTRIBUTESCLASS   REASON   AGE
+       persistentvolume/custom-modules-pvc   5Gi        RWX            Retain           Bound    newrelic/custom-modules-pvc   efs-sc         <unset>                          4m46s
 
-       1. Launch the SJM, setting a value for the `persistence.customModules` configuration value either in the command line or in a YAML file during installation. The value should specify the subpath on your synthetics job manager Persistent Volume where your custom modules files exist. For example:
+       NAME                                       STATUS   VOLUME               CAPACITY   ACCESS MODES   STORAGECLASS   VOLUMEATTRIBUTESCLASS   AGE
+       persistentvolumeclaim/custom-modules-pvc   Bound    custom-modules-pvc   5Gi        RWX            efs-sc         <unset>                 4m10s
+       ```
+
+       ### Create `mount-custom-mods-pod` to copy your custom-modules directory
+
```

Comment:
There should be a step 1 or some sort of direction here

Reply:
Thanks @jeff-colucci. It could be like:

### Custom-modules directory

1. Create `mount-custom-mods-pod` to copy your custom-modules directory